setup.py

```diff
@@ -1,9 +1,12 @@
 import os
+import site
 from distutils.core import setup
 from typing import List
 
 from setuptools import find_packages
 
+site.ENABLE_USER_SITE = True
```

Comment:
Sorry, I don't understand what is going on.

This is the first time I see something like this in `setup.py` in 10+ years of maintaining python projects.

I am 99% sure that it is not needed.

Reply:
Take a look at line 8 - line 10 in [django setup.py](https://github.com/Himanshu-Balasamanta/django/blob/main/setup.py#L8) .
In my other machine everything worked without the need of setting this flag... So I agree with you when you say that it isn't required mostly,
I think if the flag is set in the src code repo for django, the issue isn't just with my local env.